src/NodeCompiler/CompileNodeToValue.php

```diff
@@ -130,6 +130,7 @@ private function compileConstFetch(Node\Expr\ConstFetch $constNode)
      */
     private function compileClassConstFetch(Node\Expr\ClassConstFetch $node, CompilerContext $context)
     {
+        /** @var Node\Name $node->class */
```

Comment:
This syntax is not really compatible with most tooling out there - what should we do about it?

Reply:
If it breaks anything I'm happy to remove it, or replace with an `assert`.

My hope is that the PSR-5 PHPDoc proposal is approved, and the people who make similar tools add support for this. It's a common-sense annotation.

Reply:
This is also that could be fixed with a Psalm config, as it relates to behaviour of PhpParser that will never occur given how it's used in BetterReflection. https://github.com/vimeo/psalm/issues/482 would fix that internally.

Anyway, using an `assert` also fixes a PHPStan error at the same location so maybe that's the way to go here.

Reply:
Let's keep them for now 👍